go/libkb/env.go

```diff
@@ -652,11 +654,19 @@ func (e *Env) GetEmail() string {
 	)
 }
 
+// Whether to support per-user-keys in anyones sigchain.
+// Implied by UpgradePerUserKey.
+// Does not add per-user-keys to sigchains unless they are already there.
+// It is unwise to have this off and interact with sigchains that have per-user-keys.
 func (e *Env) GetSupportPerUserKey() bool {
 	if e.GetRunMode() != DevelRunMode {
 		return false
 	}
 
+	if e.GetUpgradePerUserKey() {
+		return true
+	}
```

Comment:
This ordering of checks kind of implies that in devel mode it might be possible to have "upgrade" without "support" (if we ignore the fact that GetUpgradePerUserKey has the same check order). Maybe switch them here?